### PR TITLE
Show raids loot value at the end of the raid

### PR DESCRIPTION
**Pull request by melkypie**
Adds the functionality to view raids loot value when opening the chest
image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 1]
```diff
@@ -215,6 +232,53 @@ public void onConfigChanged(ConfigChanged event)
 		clientThread.invokeLater(() -> checkRaidPresence(true));
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (!config.showLootValue() ||
+			event.getGroupId() != WidgetID.CHAMBERS_OF_XERIC_REWARD_GROUP_ID ||
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: Check widget id first, it is cheaper.

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 2]
```diff
@@ -163,6 +177,8 @@
 	@Getter
 	private boolean inRaidChambers;
 
+	private boolean chestOpened = false;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]:
```suggestion
	private boolean chestOpened;
```

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 3]
```diff
@@ -215,6 +232,53 @@ public void onConfigChanged(ConfigChanged event)
...
+		final Collection<Item> rewardItems = Arrays.stream(rewardItemContainer.getItems())
+			.filter(item -> item.getId() > 0)
+			.map(item -> new Item(item.getId(), item.getQuantity()))
+			.collect(Collectors.toList());
+
+		int rewardValue = 0;
+
+		for (Item reweardItem : rewardItems)
+		{
+			if (UNTRADABLE_ITEMS.contains(reweardItem.getId()))
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: Why is this here?

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 4]
```diff
@@ -215,6 +232,53 @@ public void onConfigChanged(ConfigChanged event)
...
+		{
+			return;
+		}
+
+		final Collection<Item> rewardItems = Arrays.stream(rewardItemContainer.getItems())
+			.filter(item -> item.getId() > 0)
+			.map(item -> new Item(item.getId(), item.getQuantity()))
+			.collect(Collectors.toList());
+
+		int rewardValue = 0;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: This should maybe be a `long`.

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java (hunk 3 above)
Comment (melkypie) [reply runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: Guess it is not needed, was thinking about it not checking item price if it is an untradable item but the item prive would be 0 anyways so it is useless, will take it out

Comment (Adam-) [reply runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: The item manager has some logic to apply prices to untradeables, but if you don't want it to there is a `ignoreUntradeableMap` flag.

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java (hunk 4 above)
Comment (melkypie) [reply runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: QuantityFormatter.quantitotyToRsDecimalStack requires an int, that is why I left it as int

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java (hunk 3 above)
Comment (melkypie) [reply runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: Removed the check
